### DISCUSSION TITLE
內捲的終結

### DISCUSSION THREAD

**Discussion by Xunhaoz** [1101-GS4719-C/Python-Community-GS4719-C#1334] [Ideas]
最近很火的提示詞工程師，聽上去是一個非常脆弱的職業，因為這個崗位根本不具備任何專業技能壁壘。曾經的隔板已經被拆除，大家都會自然語言，憑什麼你敢號稱工程師？

拿繪畫來說，一個繪畫小白和頂級插畫師之間隔著十年苦功，但一個繪畫小白之間和頂級AI畫師之間只隔了十天鑽研。同樣的，一個提示詞小白和頂級提示詞工程師，在“外圍技術”差距也幾乎可以忽略不計，真正差距其實來自創意、洞察力和品味這些“價值內核”。一個極有創意的小白，可以在短時間能掌握關鍵技術，進而生成出水平很高的作品。

因為“難度”是價值的重要構成部分。一個創作者需要克服困難，才能將某種複雜-無序轉化為簡潔、華麗、優美之類能被我們理解和接受的東西，整個過程中，“難度”是最直觀也最容易被評價的維度。即便有些“難度”本身毫無意義，比如各種奇葩的吉尼斯紀錄，它依然有某種價值。所有的“鄙視鏈”都可以在“難易程度”上得到解釋——古典和爵士就是比民謠說唱難，長篇史詩就是比短篇雜文難。

難度與專業壁壘是分工的前提，也是構成價值的重要維度，漫長的學習週期成了職業的護城河，但人工智能會把所有的護城河與壁壘變成小橋流水人家。

拿我自己的工作來舉例。我一個具有啟發性的觀點，可能幾十個字就能說清楚，但為了讓這個觀點被理解，我需要提供論證，圍繞它搭建幾千字的配套。為了讓這幾千字易於接收，我要花一天時間錄像，為了讓這種接收的效率更高，還要花三天時間做後期剪輯，最後我居然還要花一小時想標題，花兩小時排版在公眾號上發布……這裡只有幾十字是內核，其他全是外圍。

換句話說，整個創作過程，我只有兩成時間在當直人，剩下八成時間多多少少都在“犯間”，而且是越來越“間”，但由於每一個環節都有學習成本，組合起來卻成了我的“競爭力”。自媒體已經屬於相對有價值感且自由的工作了，相比之下，某些工作完全配得上“毫無意義的狗屁工作”的稱號，但那些工作居然也是有“門檻”的。這個世界上充斥著大量低價值-高門檻的荒唐工作，人們被迫寄生於這些工作中，這是一種歷史性尷尬。

有學者將電時代的技術變革稱為技能偏向型技術變革（ Skill-biased technical change），將智能時代的技術變革稱為天賦偏向型技術變革（Talent-biased technical change）（Erik Brynjolfsson, Andrew Macfee - The Second Machine Age），因為可供棲身的外圍越來越少了。

如果未來GPT開放個人API接口，我就可以調教自己的機器學徒或孿生數字人。我可以把《看本質》的全部文本餵給它，給它幾個點子和一些方向性的建議，閒聊一個下午，它就很有可能寫出一篇60甚至80分的內容。但剩下那40-20分的差距，是那些只能用風格、品味、手感、閱歷或者天賦來指稱的東西，憑著那些東西和節省出來的時間，我有可能把文章修到100分的水平。

在可見的未來，外圍技能會逐漸失去價值，內核價值被無限放大。放大到什麼程度？放大到內核本身不再稀缺以至於失去商業價值的程度。去Midjourney社區待上半天，你會知道我在說什麼。其他行業也不能倖免，你完全可以拿我的文章去餵自己的學徒，然後調教出自己的風格。

如果你的眼光足夠長遠，在未來，“工作”和“商業價值”將成為一個古老的概念，無所實指，只作修辭，就像今天的“奴隸”與“封建”一樣。

一些技術樂觀派，比如麻省理工的經濟學家布萊恩約弗森（Erik Brynjolfsson）提出了著名的數字雅典（Digital Athens）的說法：

- 古代雅典公民之所以能擁有悠閒的生活，享受民主、藝術和遊戲，主要是因為他們蓄養奴隸來做苦工。那麼，為何不用人工智能來代替奴隸，創造出一個人人都有權享受的數字化烏托邦呢？ （Max Tegmark《生命3.0》）

數字雅典也許很遙遠，不妨看看今天的希臘。據說希臘每年要上演1400多部戲劇，平均一天4部，演職人員可能比觀眾還多，搞藝術的人比欣賞藝術的人還多，這很誇張，他們賺錢嗎？不賺，有競爭嗎？沒有，就是玩。這就是人類的未來——好一點的那個版本。

**Comment by boson13579** [comment]
首先，我認為在討論職業和技能時，我們需要明確一些概念和觀點。雖然人工智能技術的發展可能改變了某些職業的面貌，但對於一個職業是否具備專業技能壁壘，仍然取決於該職業的本質和所需的知識和技能。

**Comment by boson13579** [comment]
在你提到的提示詞工程師的例子中，雖然人工智能可以幫助生成高質量的提示詞，但要成為頂級提示詞工程師，仍然需要創意、洞察力和品味等核心素質。技術的進步可能會改變工作方式和工具，但對於那些真正有創意和影響力的工作，這些技術仍然是輔助工具，而非完全取代人類的存在。

**Comment by boson13579** [comment]
此外，我同意你提到的“困難度”和“價值”的關系。克服困難並將其轉化為有意義的產物，確實可以增加作品的價值。然而，價值也不僅僅取決於技術上的困難程度，還包括創造力、洞察力、品味和與受眾的共鳴等方面。技術的發展可能會減少一些外圍技能的需求，但對於那些核心價值所在的領域，人類的創造力和獨特性仍然是無法替代的。

**Comment by boson13579** [comment]
對於未來，我認為技術的進步將繼續改變職業和工作的面貌，但這也將為人類創造更多的機會和可能性。在面對這些變化時，我會積極適應和學習新技術，同時繼續培養自己的創造力、洞察力和品味。我會保持對社會和文化的關注，通過深入研究和思考，探索那些無法被技術所替代的核心價值，並將其應用於我的工作和創作中。

**Comment by boson13579** [comment]
最重要的是，我會持續提醒自己在技術進步的浪潮中保持人類的獨特性和創造力，以及對於思考、探索和創新的重要性。無論技術如何發展，人類的智慧和創意將始終是推動社會進步和個人成長的重要因素。

**Comment by brian431** [comment]
你的這篇文章提及「我的工作」，請問同學你的工作是什麼呢？照內容看起來是跟提供觀點有關的。

**Comment by 710388** [comment]
肯定就是躺平吧，就像是大陸現在的樣子，一堆人選擇躺平

**Comment by NcuJerryKao** [comment]
你提出的觀點非常有趣，關於提示詞工程師和外圍技能的討論確實引人深思。讓我來進一步探討一下你所提到的幾個關鍵點。

外圍技能與內核價值：
你指出了外圍技能和內核價值之間的區別。外圍技能通常是可以學習和掌握的，而內核價值則涉及創意、洞察力和品味等無法僅靠技術掌握的層面。這種區別在許多領域都成立，不僅限於提示詞工程師。
像你所說的，即使AI可以生成高質量的內容，但仍然需要人類的創意和洞察力來提升作品的價值。這種結合可以產生更有深度和影響力的內容。
難度與價值：
你提到“難度”是價值的一個重要構成部分。這是一個有趣的觀點。即使某些“難度”本身毫無意義，它們仍然具有某種價值。這也解釋了為什麼我們對那些克服困難的作品更加欣賞。
在提示詞工程師的例子中，儘管技術門檻降低，但創造出有洞察力、引人入勝的提示詞仍然需要一定的難度。
未來的趨勢：
你提到外圍技能可能會逐漸失去價值，而內核價值將被放大。這是一個有趣的展望。隨著技術的發展，我們可能會看到更多價值集中在創意、洞察力和品味等內核層面。
你的想法關於使用AI調教機器學徒或孿生數字人也很有趣。這樣的結合可能會在未來產生更多令人驚艷的內容。
總之，你的觀點提醒我們，技術的進步不僅僅是技能的提升，還涉及到價值觀、創意和人性。謝謝你分享這些深入的思考！🌟
